tools/launch_benchmark_suite_scheduler_slurm.py

```diff
@@ -0,0 +1,82 @@
+import sys
```

Comment:
nit: how about renaming it to: "benchmark_suite_scheduler.py" ?

Reply:
This is slurm specific, so I think it should have slurm in the name. We will have something analagous for internal, so it would be good to be able to separate..